### PR TITLE
feat: ability to suggest bounties via tribes

### PR DESCRIPTION
**Pull request by thelostone-mc**
#### Description

**TRIBE MEMBER**
- Default message if no bounties requested
- Post Bounty requests
- Auto add as tribe member

**TRIBE OWNER** 
- Default message if priority is not set
- Default message if no bounties requested
- Update priority + publish to townsqaure
- Fund/ Reject Bounty requests

Also, introduces 2 new REST Endpoints with validation for bounty requests

### CODE REVIEW

File: app/dashboard/helpers.py [hunk 1]
```diff
@@ -389,6 +390,13 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
     else:
         raise UnsupportedSchemaException('No webReferenceURL found. Cannot continue!')
 
+    try:
+        bounty_request = BountyRequest.objects.get(github_url=url, status='o')
+        bounty_request.status = 'f'
```
Comment (octavioamu) [inline app/dashboard/helpers.py]: what is that status 'f' ?

File: app/dashboard/templates/profiles/profile.html [hunk 2]
```diff
@@ -47,7 +47,7 @@
       {% endif %}
     </div>
 
-    <div class="container-fluid bg-light">
+    <div class="container-fluid">
```
Comment (octavioamu) [inline app/dashboard/templates/profiles/profile.html]: removing bg-light will turn the bg white right?

File: app/dashboard/views.py [hunk 3]
```diff
@@ -4302,66 +4311,118 @@ def tribe_leader(request):
...
                     'is_my_org': False,
+                    'message': 'this operation is permitted to tribe owners only'
                 },
-                status=401)
+                status=405
+            )
+
+        if request.POST.get('tribe_description'):
 
+            tribe_description = clean(
```
Comment (octavioamu) [inline app/dashboard/views.py]: cool I forgot to make this changes

File: app/dashboard/templates/profiles/profile.html (hunk 2 above)
Comment (thelostone-mc) [reply app/dashboard/templates/profiles/profile.html]: Yes

File: app/dashboard/helpers.py (hunk 1 above)
Comment (thelostone-mc) [reply app/dashboard/helpers.py]: status f -> is funded

**Pull request merged**